src/imitation/algorithms/preference_comparisons.py

```diff
@@ -189,17 +189,12 @@ def __init__(
         # of `venv` when interacting with `algorithm`.
         algo_venv = self.algorithm.get_env()
         assert algo_venv is not None
-        policy_callable = rollout._policy_to_callable(
-            self.algorithm,
-            algo_venv,
-            # By setting deterministic_policy to False, we ensure that the rollouts
-            # are collected from a deterministic policy only if self.algorithm is
-            # deterministic. If self.algorithm is stochastic, then policy_callable
-            # will also be stochastic.
-            deterministic_policy=False,
-        )
+        # This wrapper ensures that rollouts are only collected from a deterministic
+        # policy (when not in exploration mode) if self.algorithm is deterministic.
+        # If self.algorithm is stochastic, then self.exploration_wrapper will also be
```

Comment:
I found this a bit confusing -- isn't `self.exploration_wrapper` *always* stochastic? It's also a bit apropos of nothing now we don't have the `deterministic_policy` keyword argument to anchor this.

I think we can probably simplify this? E.g. a rough attempt:
```
# This wrapper collects rollouts from a mixture of `self.algorithm` and a policy that acts randomly.
# The samples from `self.algorithm` are themselves stochastic if `self.algorithm` is stochastic.
# Otherwise, they are determinsitic, and action selection is only stochastic when sampling from the random policy.
```

Reply:
Your version is better - will slightly modify and go with that.